Packs/Base/ReleaseNotes/1_33_45.md

```diff
@@ -0,0 +1,5 @@
+#### Scripts
+
+##### SanePdfReports
+
+- Fixed an issue where script flag arguments treated as bool instead of string when using them as internal command arguments.
```

Comment:
```suggestion
Fixed an issue where script flag arguments were treated as boolean instead of string when using them as internal command arguments.
```